### DISCUSSION TITLE
DRH Edge Questions

### DISCUSSION THREAD

**Discussion by Ekwillj** [diabetes-research/help-desk-non-phi#5] [Q&A]
This discussion is for questions and suggestions on DRH's Edge tool. Please review the following questions below.

1. How will we handle ingested fields that are identical, but have different field names than the file format?
2. According to the id format, are id's (study_id, participant_id, site_id, etc.) automatically generated?
3. How does the platform handle missing data (fields and tables)? 
4. Will data be listed as NULL, blank, and/or 0?
5. FAQ page --We should add a section to inform researchers on how they can deidentify different columns in their files.

**Comment by razakpm** [comment]
This question is split into three parts and added as issues below:

https://github.com/diabetes-research/help-desk-non-phi/discussions/6
https://github.com/diabetes-research/help-desk-non-phi/discussions/7
https://github.com/diabetes-research/help-desk-non-phi/discussions/8
